src/app/components/SubmitButton/SubmitButton.tsx

```diff
@@ -0,0 +1,28 @@
+import type { ReactNode } from 'react';
+import React from 'react';
+import styled from 'styled-components';
+
+type ButtonProps = {
+  children: ReactNode;
+  onClick?: () => void;
+};
+export default function SubmitButton({
+  children,
+  onClick,
+}: ButtonProps): JSX.Element {
+  return <Button onClick={onClick}>{children}</Button>;
```

Comment:
I would expect that a `SubmitButton` has always `type="submit"`
```suggestion
  return <Button type="submit" onClick={onClick}>{children}</Button>;
```